jest.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
   coverageThreshold: {
     global: {
       statements: 50,
-      branches: 50,
```

Comment:
please dont' change the thresholds. Identify where we need more tests and add them

Reply:
Understood, I was unaware of the concept so I twigged it a little, will keep in mind from now on, my Next PR has all default jest.config.js configurations

Reply:
I have updated all the tests and set the jest coverage back to its default configurations. PR is ready to merge